MAINTAINERS.md

```diff
@@ -1,10 +1,13 @@
-## Maintainers
-| Maintainer | GitHub ID | Affiliation |
-| --------------- | --------- | ----------- |
-| Ashish Agrawal | [lezzago](https://github.com/lezzago) | Amazon |
-| Bowen Lan | [bowenlan-amzn](https://github.com/bowenlan-amzn) | Amazon |
-| Drew Baugher | [dbbaughe](https://github.com/dbbaughe) | Amazon |
-| Mohammad Qureshi | [qreshi](https://github.com/qreshi) | Amazon |
-| Sriram Kosuri | [skkosuri-amzn](https://github.com/skkosuri-amzn) | Amazon |
+## Overview
 
-[This document](https://github.com/opensearch-project/.github/blob/main/MAINTAINERS.md) explains what maintainers do in this repo, and how they should be doing it. If you're interested in contributing, see [CONTRIBUTING](CONTRIBUTING.md).
+This document contains a list of maintainers in this repo. See [opensearch-project/.github/RESPONSIBILITIES.md](https://github.com/opensearch-project/.github/blob/main/RESPONSIBILITIES.md#maintainer-responsibilities) that explains what the role of maintainer means, what maintainers do in this and other repos, and how they should be doing it. If you're interested in contributing, and becoming a maintainer, see [CONTRIBUTING](CONTRIBUTING.md).
+
+## Current Maintainers
+
+| Maintainer       | GitHub ID                                         | Affiliation |
+| ---------------- | ------------------------------------------------- | ----------- |
+| Ashish Agrawal   | [lezzago](https://github.com/lezzago)             | Amazon      |
+| Bowen Lan        | [bowenlan-amzn](https://github.com/bowenlan-amzn) | Amazon      |
+| Drew Baugher     | [dbbaughe](https://github.com/dbbaughe)           | Amazon      |
+| Mohammad Qureshi | [qreshi](https://github.com/qreshi)               | Amazon      |
+| Sriram Kosuri    | [skkosuri-amzn](https://github.com/skkosuri-amzn) | Amazon      |
```

Comment:
This list really doesnt match the maintainers for JS.

Reply:
@joshpalis do you happen to have the latest list?

Reply:
It's the current list in the file though. Can we baseline the list, align CODEOWNERS and update separately?

Reply:
I believe this list is missing me and @dbwiddis . Ill go ahead and update the `Maintainers.md`

Reply:
All I am saying is that it *was* missing you two. Can we merge this PR which just changes the format and then ya'll can go and fix the list in a subsequent PR?

Reply:
That makes sense, we dont have to block this, @joshpalis do you mind sending out a PR?